diff --git a/ecasound/Makefile.am b/ecasound/Makefile.am
index 7346a51d5b94..1223f346aff7 100644
--- a/ecasound/Makefile.am
+++ b/ecasound/Makefile.am
@@ -1,4 +1,4 @@
-INCLUDES = -I$(ECA_S_READLINE_INCLUDES) -I$(srcdir) -I$(top_srcdir) -I$(top_srcdir)/libecasound -I$(top_srcdir)/kvutils
+INCLUDES = -I=/usr/include/readline -I$(srcdir) -I$(top_srcdir) -I$(top_srcdir)/libecasound -I$(top_srcdir)/kvutils

 if ECA_AM_USE_NCURSES
 termcap_library_ncurses = -lncurses
